tests/core/models/test_edition.py

```diff
@@ -325,6 +328,67 @@ def test_calculate_presentation_author(self, db: DatabaseTransactionFixture):
         assert "Kelly Accumulator, Bob A. Bitshifter" == wr.author
         assert "Accumulator, Kelly ; Bitshifter, Bob" == wr.sort_author
 
+    def test_calculate_presentation_very_long_author(
+        self, db: DatabaseTransactionFixture
+    ):
+        authors = []
+
+        # author names should be unique and not similar to ensure that the
+        # test mirrors the types of long author lists we'd expect in real data.
+        def generate_random_random_author():
+            return "".join(
+                random.choices(
+                    string.ascii_uppercase + string.ascii_lowercase + string.digits,
```

Comment:
Minor, but do we want digits in the random author names? Or are we just adding some more randomness here?

Reply:
I wanted to add maximal randomness.

Reply:
Could be handy to add a comment to that effect.